src/Microsoft.Identity.Client/Features/ConfidentialClient/ConfidentialClientApplication.cs

```diff
@@ -397,9 +397,9 @@ async Task<AuthenticationResult> IConfidentialClientApplicationWithCertificate.A
         /// From there you will be able to use MSAL.NET new API, in particular
         /// AcquireTokenSilentAsync() which will renew the user token.  
         /// </summary>
-        async Task<AuthenticationResult> IByRefreshToken.AcquireTokenByRefreshTokenAsync(string refreshToken)
+        async Task<AuthenticationResult> IByRefreshToken.AcquireTokenByRefreshTokenAsync(IEnumerable<string> scopes, string refreshToken)
```

Comment:
@jmprieur Scopes parameter here that can be set to null. is this API ok?

Reply:
sure

Reply:
You should add a logging statement for this, e.g. log the number of scopes

Reply:
done